build-logic-commons/build-platform/build.gradle.kts

```diff
@@ -29,7 +29,7 @@ dependencies {
         api("com.gradle.publish:plugin-publish-plugin:1.2.1")
         api("gradle.plugin.org.jetbrains.gradle.plugin.idea-ext:gradle-idea-ext:1.0.1")
         api("me.champeau.gradle:japicmp-gradle-plugin:0.4.1")
-        api("me.champeau.jmh:jmh-gradle-plugin:0.6.8")
+        api("me.champeau.jmh:jmh-gradle-plugin:0.7.2")
```

Comment:
I assume the jmh plugin removed some `Project.getConventions` calls in this release.

Reply:
This was about another deprecation - how to create a configuration.